bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java

```diff
@@ -170,7 +162,7 @@ private void updateAlert(String channelId, AlertLevel value) {
         }
         if (isLinked(channelIcon)) {
             State result = UnDefType.UNDEF;
-            byte[] bytes = getResource(String.format("picto/%s.svg", channelId));
+            byte[] bytes = getResource(String.format("picto/%s.svg", channelId.replace("-", "_")));
```

Comment:
PR openhab/openhab-core#3548 will be merged very soon and remove the constraint with hyphen.
So no need to do these replaces.

Reply:
Meaning I can keep hyphens in the icon name and it won't hurt ?

Reply:
Yes, as soon as the PR will be merged, that is just after milestone 2 is released.

Reply:
Done